SimTracker/TrackTriggerAssociation/plugins/TTTrackAssociator.h

```diff
@@ -49,12 +49,12 @@ class TTTrackAssociator : public edm::stream::EDProducer<> {
 
 private:
   /// Data members
-  std::vector<edm::InputTag> TTTracksInputTags;
+  std::vector<edm::InputTag> ttTracksInputTags;
```

Comment:
member variables should end in underscore

Reply:
done